sstables/types.hh

```diff
@@ -129,7 +129,7 @@ inline std::ostream& operator<<(std::ostream& o, indexable_element e) {
 
 class summary_entry {
 public:
-    dht::token_view token;
+    dht::token token;
```

Comment:
why?

Reply:
Maybe it'd make more sense to just remove the `token_view` class now. The reason this class exists is that it's trivially destructible, so it's easy to dispose giant vectors holding `token_view`s.
But now `token` is trivially destructible. (I'd suggest to add a `static_assert` to check this)